ONSdigital/eq-author
# Modal with Navigation/Tabs

### What is the context of this PR?

[Trello card](https://trello.com/c/qdAqh5Gd/466-validation-modal)

<img width="1011" alt="screenshot 2018-07-20 09 19 12" src="https://user-images.githubusercontent.com/930398/42991598-ffb0d4ac-8bfd-11e8-81ca-596177d658e2.png">


Adds a Modal with tabs navigation for use with validation.

- new Tabs component with Redux implementation—this can be used for any generic tabs implementation although we'd have to look at renaming/moving the existing Tabs component.
- ModalWithNav implement Modal with a Grid layout and Tabs
- content of modal out of scope of this story
- Tabs use [this example](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html) for aria implementation

### How to review 

- visit storybook entry
- confirm navigation works as expected
